app/javascript/gobierto_dashboards/webapp/lib/config.js

```diff
@@ -0,0 +1,12 @@
+export const contractsColumns = [
```

Comment:
Should we move the columns to the module configuration JSON?

Reply:
Is this JSON config already available from de API? Our initial idea was fetch the site metadata from XHR call (because it should be stored in DB), instead having a json config file in the code